src/index.js

```diff
@@ -1,17 +1,24 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter, Switch, Route, Link } from 'react-router-dom';
+import 'bootstrap/dist/css/bootstrap.min.css';
 import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import Main from './components/main';
+
+const App = () => (
+  <main>
+    <h1 className="text-center">Contact List</h1>
+    <Switch>
+      <Route path="/" component={Main} />
+    </Switch>
```

Comment:
This is redundant given the contents of main - you could simply have the three routes from main here